x-pack/filebeat/module/ibmmq/errorlog/test/AMQERR01.log

```diff
@@ -0,0 +1,356 @@
+                      Host(FELIX-ELASTIC) Installation(Installation1)
+                      VRMF(9.1.0.0)
+                      Time(2018-10-28T14:12:07.685Z)
+                      ArithInsert1(73)
+                     
+AMQ7125I: There are 73 days left in the trial period for this copy of IBM MQ.
+
+EXPLANATION:
+This copy of IBM MQ is licensed for a limited period only.
+ACTION:
+None. 
+----- amqzlic0.c : 435 --------------------------------------------------------
+28/10/2018 15:12:07 - Process(7144.1) User(MUSR_MQADMIN) Program(strmqm.exe)
+                      Host(FELIX-ELASTIC) Installation(Installation1)
+                      VRMF(9.1.0.0)
+                      Time(2018-10-28T14:12:07.789Z)
+                      ArithInsert1(73)
+                     
+AMQ7125I: There are 73 days left in the trial period for this copy of IBM MQ.
+
+EXPLANATION:
+This copy of IBM MQ is licensed for a limited period only.
+ACTION:
+None. 
+----- amqzlic0.c : 435 --------------------------------------------------------
+28/10/2018 15:12:08 - Process(7360.1) User(MUSR_MQADMIN) Program(amqzxma0.exe)
+                      Host(FELIX-ELASTIC) Installation(Installation1)
+                      VRMF(9.1.0.0)
+                      Time(2018-10-28T14:12:08.663Z)
+                     
+AMQ6776E: Queue manager recovery log not correctly configured for advanced
+format disk.
+
+EXPLANATION:
+The queue manager recovery log is located on an advanced format disk, however
+the recovery log is not configured correctly to provide full write integrity
+with advanced format disks. Advanced format disks require 4KB alignment for
+reliable writing, but the current queue manager was created at an MQ version
+that did not yet support advanced format disks and did not enforce 4KB
+alignment.
+ACTION:
+MQ has only supported placing the recovery log on an advanced format disk since
+MQ V9.0.4. MQ V9.0.4 introduced the MIGMQLOG command that can be used to
+migrate a pre-existing queue manager recovery log to a format suitable for
+advanced format disks. Any queue manager created on a version of V9.0.4 or
+higher will have been created with a recovery log suitable for advanced format
+disks. Investigate migrating the queue manager recovery log, using the MIGMQLOG
+command, to a format suitable for the current log location. 
+----- amqxfdcp.c : 832 --------------------------------------------------------
+28/10/2018 15:12:08 - Process(7360.1) User(MUSR_MQADMIN) Program(amqzxma0.exe)
+                      Host(FELIX-ELASTIC) Installation(Installation1)
+                      VRMF(9.1.0.0)
+                      Time(2018-10-28T14:12:08.665Z)
+                      ArithInsert1(7360)
+                      CommentInsert1(QM1)
+                     
+AMQ6184W: An internal IBM MQ error has occurred on queue manager QM1.
+
+EXPLANATION:
+An error has been detected, and the IBM MQ error recording routine has been
+called. The failing process is process 7360.
```

Comment:
@ruflin Yeah I think that "process" may be pid + thread_id. Here, the error message refers to "process 7360", and the structured message above contains "7360.1".